### ISSUE TITLE
Accessibility: datatable with row expander contains anchor tag must have discernible text

### ISSUE THREAD

**Issue by manidevzen3** [primefaces/primeng#7069]
[x] bug report

**Current behavior**
anchor element in expander rows doesn't have visible text or labels

**code snippet**
_<a href="#">
    <span class="ui-row-toggler fa fa-fw ui-clickable fa-chevron-circle-down"></span>
</a>_

**Expected behavior**
anchor elements should have relevant text or labels to be able to read by screen readers

* **Angular version:** 4.1.3

* **PrimeNG version:** 4.2.1

**Comment by cagataycivici** [comment]
Unable to replicate with PrimeNG 9, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
